### ISSUE TITLE
Screenreader add global search and cancel for global search

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#5602]
**Current behavior**
The search icon from the global search is missing the ARIA-label and the global search dialog is not focusing the cancel button (and hence not reading it out loud)

**Wanted behavior**
- The global search icon should have an ARIA-label
- The cancel button from the global search should be focusable

**Additional:**
Focusable mean that if you use tap to navigate through a page the tap will at some point reach that place.

The global search:
<img width="878" height="95" alt="Image" src="https://github.com/user-attachments/assets/d7333fb1-50e9-4154-b7ea-3772ef51958c" />

The cancel button is the X in the corner
(Before taking the screenshot I hovered over the icon, that's why it seems focused, the shadow indicates focus a lot of the times)
<img width="769" height="256" alt="Image" src="https://github.com/user-attachments/assets/10137318-5100-4a1a-87c2-0b7d3cb237b6" />
